addons/account/tests/test_account_move_entry.py

```diff
@@ -448,9 +448,10 @@ def test_included_tax(self):
             'price_include': True,
             'include_base_amount': False,
         })
-        self.account = self.env['account.account'].search([('deprecated', '=', False)], limit=1)
+        self.account = self.company_data['default_account_receivable']
 
         move_form = Form(self.env['account.move'])
```

Comment:
`move_form = Form(self.env['account.move'].with_context(default_move_type='entry')))`
to get a default journal (remove the above diff then).